add "decode" option to panel (#2284)
kyma-project/busola: docs/extensibility/display-section.md

- **header** - an optional array that allows you to, for example, display labels in the panel header.
- **disablePadding** - an optional boolean which disables the padding inside the panel body.
- **description** - displays a custom description on the resource list page. It can contain links. If the **translations** section has a translation entry with the ID that is the same as the **description** string, the translation is used.

> - **decodable** - decodes the values of all the children using base64 - has to be used together with the **source** parameter.

Reviewer: ```suggestion
- **decodable** - decodes the values of all the children using base64 - must be used together with the **source** parameter.
```
